### ISSUE TITLE
Frosted Glass Promos

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#9667]
REF: https://jira.dev.bbc.co.uk/browse/NEWSWORLDSERVICE-1375

Depends on: https://github.com/bbc/simorgh/issues/9666 (probably)

### Task
Create High-Impact promo for the Features component of a CPS page/Optimo page
The promo should be able to replace any promo in the Features component.

The promo will support the following promo types:
- Articles

Media and picture gallery indicators will not be supported for the first iteration

 **Acceptance Criteria:**
- The promo will be visible as part of the Features and Analysis onward journeys on CPS STY and Optimo story pages
- The high impact promo will support both AMP and canonical
- The high impact promo will need to support RtoL and LtoR services
- The promo will be able to replace any promo within the Features component.
- Promo requires fallback colour where browsers are not supported
- Promo requires fallback colour where JS has been disabled
- Promo will need to support event tracking 

  
**UX**

Design scenarios: https://paper.dropbox.com/doc/High-impact-promos-colour-logic--BWeOmqRoT34Fx1l2Qhlnm85SAQ-NyPlhnx01lbpgZxSXHR3p

High impact article promo: Blur, dynamic colour (modern browser) — dark vibrant
https://zpl.io/VD30DdW
High impact article promo: Blur, no dynamic colour — Chameleon grey 8 62%
https://zpl.io/V1LBQYB
High impact article promo: No blur, no dynamic colour (IE, JS disabled) — solid Chameleon grey 8
https://zpl.io/2ZANDoX
High impact article promo: No blur, dynamic colour (IE) — solid dark vibrant
https://zpl.io/VYzEXqk

Prototype can be found here: 
https://5d28eb3fe163f6002046d6fa-jmtbbocxpt.chromatic.com/?path=/story/fancy-promos--so-fancy:

**A11y A/C:**
https://paper.dropbox.com/doc/High-Impact-Promo-a11y-criteria--BWTs1kBFJRU1kvBloBEVsqmVAQ-Nr6QNmdYhBOioeqf62PID

Screenreader UX: (should not be changed from current screenreader UX)
https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/613a12ef4bc12cba0f52d609
